packages/services/queue-bull/src/bull-queue-service.ts

```
@@ -0,0 +1,115 @@
+import { mapBullJobToQueueJob } from './utils.js'
+
+/**
+ * Bull/Redis queue service implementation
+ * Supports job results, retries, and most queue features
+ */
+export class BullQueueService implements QueueService {
+  readonly name = 'bull'
+  readonly supportsResults = true
+  protected queues = new Map<string, Bull.Queue>()
+
+  constructor(_redisConfig: Bull.QueueOptions | undefined) {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)